MergePythonSDK/accounting/model/common_model_scopes_body_request.py

```diff
@@ -0,0 +1,283 @@
+"""
+    Merge Accounting API
+
+    The unified API for building rich integrations with multiple Accounting & Finance platforms.  # noqa: E501
+
+    The version of the OpenAPI document: 1.0
+    Contact: hello@merge.dev
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import re  # noqa: F401
```

Comment:
I still dont understand how this is back

Reply:
talked offline since we see it in the prod schema assuming this is correct